apps/web/app/api/admin/tokens/route.ts

```
@@ -0,0 +1,63 @@
+  const auth = await requireAdmin(request);
+  if (auth instanceof NextResponse) return auth;
+
+  try {
+    const { operation, token, amount, address, memo } = await request.json();
+    
+    const auditContext = {
+      ipAddress: auth.ipAddress,
+      userAgent: auth.userAgent,
+    };
+
+    let data;
```

Biome (2.4.9)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)